src/service/index.ts

```diff
@@ -25,21 +25,30 @@ import { Node } from 'src/model/node';
 import { SocketListenerManager } from 'src/service/socketListenerManager';
 import { nodeComparator } from 'src/util/comparator/node';
 
-export const blockService = new BlockService(blockRepository);
 export const blockchainService = new BlockchainService(blockchainRepository);
 export const systemService = new SystemService(systemRepository);
 export const accountService = new AccountService();
 export const webhookService = new WebhookService<WebhookAction | EVENT_TYPES>();
-export const transactionConfirmationService = new TransactionConfirmationService(
-    transactionRepository,
-    blockRepository,
-    NUMBER_OF_CONFIRMATIONS,
-);
-export const transactionService = new TransactionService(
+export const transactionService = new TransactionCreator(
     slotService,
     timeService,
     accountService,
 );
+
+const nodes = NODE_HOSTS
+    .map(host => initSocketIOClient(host.ip, host.port))
+    .map(socket => new SocketIOClient<API_ACTION_TYPES | EVENT_TYPES>(socket))
+    .map(socket => new Node(socket));
```

Comment:
What is the point?

Reply:
The Node contains information about blockchain